src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-bottom: 100px;
```

Comment:
Consider moving the margin property to a separate utility class or a parent container to avoid applying margin directly to the button component.